fdbclient/ServerKnobs.cpp

```diff
@@ -164,6 +164,15 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( PRIORITY_SPLIT_SHARD,                                  950 ); if( randomize && BUGGIFY ) PRIORITY_SPLIT_SHARD = 350;
 
 	// Data distribution
+	init( AVAILABLE_SPACE_PIVOT_RATIO,                         0.6 );
+	init( CPU_PIVOT_RATIO,                                     0.9 );
+	// In order to make sure GetTeam has enough eligible destination team:
+	ASSERT_GT(AVAILABLE_SPACE_PIVOT_RATIO + CPU_PIVOT_RATIO, 1.0 );
```

Comment:
I am not fully understand `ASSERT_GT` here. If we set `AVAILABLE_SPACE_PIVOT_RATIO = 0.9, CPU_PIVOT_RATIO = 0.9`, there might not be many eligible teams?